src/Google/Ads/GoogleAds/Lib/V16/UnaryGoogleAdsResponseMetadataCallable.php

```diff
@@ -21,6 +21,7 @@
 use Google\Ads\GoogleAds\Lib\GoogleAdsMiddlewareAbstract;
 use Google\ApiCore\Call;
 use Google\ApiCore\Middleware\ResponseMetadataMiddleware;
+use GuzzleHttp\Promise\PromiseInterface;
```

Comment:
Could you add the import for `Promise` too?

Reply:
done!